kura/examples/org.eclipse.kura.example.ble.tisensortag/src/main/java/org/eclipse/kura/example/ble/tisensortag/BluetoothLe.java

```diff
@@ -298,170 +250,132 @@ void checkScan() {
     //
     // --------------------------------------------------------------------
 
-    protected static void doPublishKeys(String address, Object key) {
+    protected void doPublishKeys(String address, Integer key) {
         KuraPayload payload = new KuraPayload();
         payload.setTimestamp(new Date());
         payload.addMetric("key", key);
         try {
-            cloudClient.publish(topic + "/" + address + "/keys", payload, 0, false);
+            this.cloudClient.publish(topic + "/" + address + "/keys", payload, 0, false);
```

Comment:
I had the same doubt, so I did some research and I found [this](https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java).